libs/apollo-client/src/lib/apollo-client.ts

```diff
@@ -61,12 +64,34 @@ export function createClient(base?: string, cacheConfig?: InMemoryCacheConfig) {
     : httpLink;
 
   const errorLink = onError(({ graphQLErrors, networkError }) => {
-    if (graphQLErrors) console.log(graphQLErrors);
-    if (networkError) console.log(networkError);
+    if (graphQLErrors && !hasNotFoundGraphQLErrors(graphQLErrors)) {
```

Comment:
chore (blocking): If graphqlErrors contains a mix of NotFound errors and legitimate errors this check will now prevent the legitimate errors from being logged. Can we change this so we filter out any NotFound errors and still log any remaining errors?

Reply:
sure, updated in last commit.